services/venice-controller/src/main/java/com/linkedin/venice/ingestion/control/RealTimeTopicSwitcher.java

```diff
@@ -222,42 +221,63 @@ public void transmitVersionSwapMessage(Store store, int previousVersion, int nex
     Version previousStoreVersion = store.getVersionOrThrow(previousVersion);
     Version nextStoreVersion = store.getVersionOrThrow(nextVersion);
 
-    // Only transmit version swap message to RT's if there is a view config (temporary check)
-    if (!hasViewConfigs(nextStoreVersion, previousStoreVersion)) {
-      // NoOp for now
-      return;
+    // If there exists an RT, then broadcast the Version Swap message to it, otherwise broadcast it to the VT
+    String storeName = store.getName();
+    if (!topicManager.containsTopic(pubSubTopicRepository.getTopic(Version.composeRealTimeTopic(storeName)))) {
```

Comment:
I wonder if we should add a TODO here for broadcasting to batch only view topics?  I don't think we should do that with this PR just to keep things small.  There shouldn't be any user of DVRT/CDC on a view which is batch only, so then the only thing that would remain is if DVC consumption on a view topic needs this message for some reason?